packages/flutter/lib/src/cupertino/expansion_tile.dart

```diff
@@ -0,0 +1,245 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// @docImport 'package:flutter/material.dart';
+///
+/// @docImport 'list_section.dart';
+library;
+
+import 'package:flutter/widgets.dart';
+
+import 'colors.dart';
+import 'icons.dart';
+import 'list_tile.dart';
+import 'theme.dart';
+
+/// The curve of the animation used to expand or collapse the
+/// [CupertinoExpansionTile].
+///
+/// Eyeballed from an iPhone 15 simulator running iOS 17.5.
```

Comment:
Uber nit, and probably something to follow up on later. `eyeball` comments are usually `//` - not public facing docs (`///`).
I see this is mostly the case for the framework, but it looks like it's not consistent.

Reply:
Oh right, silly me I missed that was private. I guess that is what I get for doing code review after hours. :)